doc/windows-dev.txt

```diff
@@ -0,0 +1,25 @@
+Steps to have a development environment running on a windows machine
+
+1.  Install git
+2.  Install python 3.5+ from python.org
+3.  clone the project:
+    git clone https://waffle.io/HewlettPackard/oneview-redfish-toolkit
+4.  Access project directory:
+    cd oneview-redfish-toolkit
+5.  Create a virtual environment(venv):
+    py -m venv .venv
+    NOTE: .venv is the directory of the virtual environment. You can name it whatever you like, given it's not an existing dir in the project.
+6.  activate the virtual env:
+    ./venv/scripts/activate

```

Comment:
I tried this here but it only worked when using: `.\.venv\Scripts\activate.bat`
Any idea if this is specific to my windows 7 environment?

Reply:
the activate script checks if you are running command or powershell and calls activate.bat or activate.ps1 accordingly.
I've tested with windows 10 in powershell and command prompt and worked. I you want I can build a windows 7 environment. But I think it's easier just to explain and tell people about the other activate scripts. Check if after the update and tell me what you want...